components/00-settings/_colors.scss

```diff
@@ -205,6 +205,8 @@ $themes: (
     'color-primary': color('cyan'),
     'color-secondary': color('white'),
     'color-tertiary': color('dark-gray'),
+    'color-error': color('red'),
```

Comment:
These were not themeable by design to implement consistency over all themes and prevent 'artistic' interpretations of error and success colors.